### PR TITLE
fix(cxl-ui): cxl-marketing-nav menu bar button alignment

### PR DESCRIPTION
**Pull request by anoblet**
https://app.clickup.com/t/86b2hv84w

### CODE REVIEW

File: packages/cxl-ui/scss/cxl-marketing-nav.scss [hunk 1]
```diff
@@ -33,10 +33,20 @@
       .container {
         justify-content: space-between;
 
+        .cxl-logo{
```
Comment (lkraav) [inline packages/cxl-ui/scss/cxl-marketing-nav.scss:36]:
```suggestion
        .cxl-logo {
```

**Comment by anoblet** [comment]
Logo:
image
Menu:
image

**Comment by anoblet** [comment]
I noticed that his change should also apply to desktop. I opened a new PR: https://github.com/conversionxl/aybolit/pull/442.

**Comment by lkraav** [comment]
> I noticed that his change should also apply to desktop. I opened a new PR: https://github.com/conversionxl/aybolit/pull/442.

Yep exactly that's why screenshotting in PRs remains a good practice.

**Pull request merged**